### ISSUE TITLE
images not shown in sidebar

### ISSUE THREAD

**Issue by laurencefass** [formulahendry/vscode-docker-explorer#34]
i can see containers but no images. 

when i click refresh on the tab title there is no console output nor network activity.

there also appears to be no refresh option on the images tab (as there is with containers).

**Comment by jwallander** [comment]
I have the same problem. Docker images section in Docker Explorer empty. The images are shown in Docker extension.

Docker Version 18.05.0-ce-win67 (18263) 
Windows containers
with "experimental": true, "graph": "D:\\Docker"

Win 10 Pro 1803
VSC 1.24.0

**Comment by freddydk** [comment]
I have the same and I think the reason is, that I have two images with the same ID. It is of course the same image, just two different tags for the same image.
![image](https://user-images.githubusercontent.com/10775043/62359728-b0682000-b517-11e9-90df-0cc7b2407536.png)
When refreshing images, I get this error in the VS Code corner
![image](https://user-images.githubusercontent.com/10775043/62359783-c7a70d80-b517-11e9-912b-637c40fead21.png)

**Comment by NKarolak** [comment]
> the reason is, that I have two images with the same ID

True, same here (I did not discover this issue earlier, hence I created a new one): https://github.com/formulahendry/vscode-docker-explorer/issues/48

**Comment by sufyanel** [comment]
I have different tags for every version. It still gives me the same error.
